lib/pages/admin/delete_choice_board_category.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+
+/// Deletes a choiceboard category given ID
+class DeleteChoiceBoardCategory extends StatelessWidget {
+  final String categoryId;
+
+  const DeleteChoiceBoardCategory({Key? key, required this.categoryId})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    // Return to admin screen if user cancels choice
+    Widget cancelButton = TextButton(
+      child: Text("No"),
+      onPressed: () {
+        Navigator.pop(context);
+      },
+    );
+
+    // Delete document from firebase collection
+    void deleteFromCollection(String collectionName) {
+      FirebaseFirestore.instance
+          .collection(collectionName)
+          .doc(categoryId)
+          .delete()
+          .then(
+            (doc) => print("Document deleted"),
+            onError: (e) => print("Error updating document $e"),
+          );
+    }
+
+    // Once user confirms choice, call delete function
+    Widget confirmButton = TextButton(
+      child: Text("Yes"),
+      onPressed: () {
+        // Delete associated document from 'categoryItems' collection
+        deleteFromCollection("categoryItems");
+
+        // Delete category from 'categories' collection
+        deleteFromCollection("categories");
```

Comment:
The user is not sent back to admin choice boards after deleting

Reply:
I'm almost sure they are - by either tapping outside the popup, tapping cancel, or tapping delete, the popup closes. It's important to remember this isn't a new screen per se, just something overlayed on screen that is then dismissed.

Reply:
I think you're missing `Navigator.pop(context);`